MC/run/ANCHOR/2021/OCT/pass4/anchorMC.sh

```diff
@@ -84,8 +84,7 @@ NSIGEVENTS=${NSIGEVENTS:-22}
 baseargs="-tf ${NTIMEFRAMES} --split-id ${ALIEN_JDL_SPLITID:-0} --prod-split ${ALIEN_JDL_PRODSPLIT:-100} --run-number ${RUNNUMBER}"
 
 # THIS NEEDS TO COME FROM OUTSIDE
-remainingargs="-eCM 900 -col pp -gen pythia8 -proc inel -ns ${NSIGEVENTS}                                                                                                 \
-               -interactionRate 2000                                                                                                                                      \
```

Comment:
we cannot just take it way. There would be no default then. Something like `-interactionRate ${IRATE:-2000}` might be ok.

Reply:
Hi Sandro,

Ok, agreed not to remove the interaction rate (it was intended to be retrieved from CCDB, but we're not yet there). Will open a new PR, this one cannot be continued, as I have not pushed it from a branch of my fork.

Thanks!

Yours,
Catalin.